git-cliff/src/lib.rs

```diff
@@ -303,6 +279,36 @@ fn process_repository<'a>(
 			releases.push(Release::default());
 		}
 	}
+	if recurse_submodules {
+		for (_submodule, _commits) in submodule_commits.values() {
+			log::trace!("Recursing {:?}.", _submodule.path().to_str());
+			let _range = format!(
+				"{}..{}",
+				_commits.first().unwrap().id,
+				_commits.last().unwrap().id
+			);
+			let _repo_path = _submodule.path().to_path_buf();
+			let _repo = Repository::init(_repo_path.clone())?;
```

Comment:
You can get rid of the compiler error by doing this for now:

```rust
let repo = Box::leak(Box::new(Repository::init(repo_path.clone())?));
```

It's a bit bad, but this is also what I'm doing for the other repositories 💀